cmd/kueue/main.go

```diff
@@ -370,6 +371,13 @@ func waitForPodsReady(cfg *configapi.Configuration) bool {
 	return cfg.WaitForPodsReady != nil && cfg.WaitForPodsReady.Enable
 }
 
+func podsReadyRequeuingTimestamp(cfg *configapi.Configuration) configapi.RequeuingTimestamp {
+	if cfg.WaitForPodsReady != nil {
+		return cfg.WaitForPodsReady.RequeuingTimestamp
```

Comment:
If we go for ptr, I think we will also need to check it is not nil here.

Reply:
Updated to pointer and added nil check